AWSLocation.podspec

```diff
@@ -12,6 +12,6 @@ Pod::Spec.new do |s|
   s.source       = { :git => 'https://github.com/aws/aws-sdk-ios.git',
                      :tag => s.version}
   s.requires_arc = true
-  s.dependency 'AWSCore', '2.22.0'
-  s.source_files = 'AWSLocation/*.{h,m}', 'AWSLocation/AWSLocationTracker/**/*.swift'
+  s.dependency 'AWSCore', '2.22.1'
+  s.source_files = 'AWSLocation/*.{h,m}'
```

Comment:
this is a bit of a concern as we have manually modified the podspec to expose the high level AWSLocationTracker client, and the model updates are removing this change

Reply:
Keeping this PR open to keep it visible, https://github.com/aws-amplify/aws-sdk-ios/issues/3412